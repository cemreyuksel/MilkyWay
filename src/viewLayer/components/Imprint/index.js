import React from 'react';

const Imprint = () => <h2 div="imprint-header">Imprint</h2>;

export default Imprint;