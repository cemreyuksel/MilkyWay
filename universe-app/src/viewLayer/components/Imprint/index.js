import React from 'react';

export default () => <h2 div="imprint-header">Imprint</h2>;
