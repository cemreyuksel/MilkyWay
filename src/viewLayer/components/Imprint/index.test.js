import React from 'react';
import renderer from 'react-test-renderer';

import Imprint from './index';

describe('Imprint component', () => {
  test('renders', () => {
    const tree = renderer.create(<Imprint />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
