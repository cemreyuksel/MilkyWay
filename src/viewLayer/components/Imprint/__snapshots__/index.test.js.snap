// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Imprint component renders 1`] = `
<h2
  div="imprint-header"
>
  Imprint
</h2>
`;
